argo-cd-apps/overlays/konflux-public-production/kustomization.yaml

```diff
@@ -12,9 +12,6 @@ resources:
 
 namespace: konflux-public-production
 
-patchesStrategicMerge:
-  - delete-applications.yaml
-
 patches:
```

Comment:
I forgot to tell you before, you need to add a path in the list below for project-controller:

```
- path: production-overlay-patch.yaml
    target:
      kind: ApplicationSet
      version: v1alpha1
      name: project-controller
```

and you need the same in argo-cd-apps/overlays/production-downstream/kustomization.yaml

Reply:
Added